src/widget/wmainmenubar.cpp

```diff
@@ -883,8 +880,8 @@ void WMainMenuBar::slotDeveloperStatsExperiment(bool enable) {
 }
 
 void WMainMenuBar::slotDeveloperDebugger(bool toggle) {
-    m_pConfig->set(ConfigKey("[ScriptDebugger]","Enabled"),
-                   ConfigValue(toggle ? 1 : 0));
+    m_pConfig->setValue(ConfigKey(QStringLiteral("[ScriptDebugger]"), QStringLiteral("Enabled")),
```

Comment:
Same here - since `setValue<bool>` relies on integer for serialisation, this should be safe to update

Reply:
copypasta here and above.
will fix it